src/streams.h

```diff
@@ -421,7 +422,7 @@ class CDataStream
         }
 
         for (size_type i = 0, j = 0; i != size(); i++) {
-            vch[i] ^= key[j++];
+            vch[i] ^= std::byte{key[j++]};
```

Comment:
I think it would make sense to make ObfuscateKey a `std::byte` throughout, avoiding this cast. Though no need to do so in this PR.